modules/rosa-upgrading-manual-ocm.adoc

```diff
@@ -3,6 +3,12 @@
 // * rosa_upgrading/rosa-upgrading.adoc
 // * rosa_upgrading/rosa-upgrading-sts.adoc
 
+ifdef::openshift-rosa[]
+ifeval::["{context}" == "rosa-hcp-upgrading"]
+:hcp-in-rosa:
+endif::[]
+endif::openshift-rosa[]
```

Comment:
Added for the specific scenario where there is an HCP-only procedure that is appearing in the `openshift-rosa` distro.

I.e., the `rosa-hcp-upgrading` assembly appears in the `openshift-rosa` distro in this case. This temporary `hcp-in-rosa` entity allows using `ifdef`/`ifndef` to target that situation.